src/atomate2/lobster/jobs.py

```diff
@@ -0,0 +1,90 @@
+"""Module defining lobster jobs."""
+
+from __future__ import annotations
+
+import logging
+from dataclasses import dataclass, field
+from pathlib import Path
+
+from jobflow import Maker, job
+from monty.shutil import gzip_dir
+from pymatgen.electronic_structure.cohp import CompleteCohp
+from pymatgen.electronic_structure.dos import LobsterCompleteDos
+from pymatgen.io.lobster import Lobsterin
+
+from atomate2.lobster.files import copy_lobster_files
+from atomate2.lobster.run import run_lobster
+from atomate2.lobster.schemas import LobsterTaskDocument
+
+__all__ = ["PureLobsterMaker"]
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class PureLobsterMaker(Maker):
+    """
+    LOBSTER job maker.
+
+    Parameters
+    ----------
+    name : str
+        Name of jobs produced by this maker.
+    resubmit : bool
+        Maybe useful.
+    task_document_kwargs : dict
+        Keyword arguments passed to :obj:`.LobsterTaskDocument.from_directory`.
+    """
+
+    name: str = "lobster"
+    resubmit: bool = False
+    task_document_kwargs: dict = field(default_factory=dict)
+
+    @job(output_schema=LobsterTaskDocument, data=[CompleteCohp, LobsterCompleteDos])
+    def make(
+        self,
+        wavefunction_dir: str | Path = None,
+        basis_dict: dict | None = None,
+        user_lobsterin_settings: dict | None = None,
+        additional_outputs: list[str] | None = None,
```

Comment:
I'm just wondering about the motivation for putting `user_lobsterin_settings` and `additional_outputs` here rather than as fields on the Maker class? Normally, I'd put arguments in the `make` function if they will change for each job (e.g., depending on the structure or basis set). But it seems like these two parameters are reasonably fixed. 

If you define them on this Maker class, this would allow you to remove them from the Flow maker class and avoid duplication of parameters. 

Just a thought, happy to keep it if you prefer it this way round.

Reply:
Thanks for the suggestion. You are probably correct that this would reflect the overall logic of atomate2 much better. I have to check if I can implement it without breaking some functionality of the workflow again.

Reply:
Implemented.